### PR TITLE
(RE-9130) Add Debian 9 (stretch) platform config

### PR DESCRIPTION
**Pull request by mwaggett**

### CODE REVIEW

**Comment by mwaggett** [comment]
I just copied from the debian 8 config.

**Comment by ScottGarman** [comment]
Technically stretch (and jessie for that matter) uses systemd and not init.d. But that doesn't really matter for this particular package.

**Comment by melissa** [comment]
I suggest you copy the file from the PC1 branch for consistency

**Comment by melissa** [comment]
https://github.com/puppetlabs/puppetlabs-release/blob/PC1/configs/platforms/debian-9-amd64.rb

**Comment by mwaggett** [comment]
@ScottGarman @melissa Updated to use systemd for stretch and jessie. I copied form the PC1 config, but also left the `plat.add_build_repository` bit.

File: configs/platforms/debian-9-amd64.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+platform "debian-9-amd64" do |plat|
+  plat.servicedir "/lib/systemd/system"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "systemd"
+  plat.codename "stretch"
+
+  plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/debian/pl-build-tools-release-stretch.deb"
```
Comment (melissa) [inline configs/platforms/debian-9-amd64.rb]: You should take this line out. I've been slowly phasing them out, hence why it wasn't in the PC1 config.

Comment (ScottGarman) [inline configs/platforms/debian-9-amd64.rb]: Yeah, that will cause problems for folks outside our network who don't have access to the pl-build-tools server.

Comment (melissa) [inline configs/platforms/debian-9-amd64.rb]: plus, the release package don't use anything that is available in the pl-build-tools repos, so there's really no reason to have them.

**Comment by mwaggett** [comment]
@melissa updated!

**Comment by ScottGarman** [comment]
:+1:

File: configs/platforms/debian-8-amd64.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 platform "debian-8-amd64" do |plat|
-  plat.servicedir "/etc/init.d"
+  plat.servicedir "/lib/systemd/system"
   plat.defaultdir "/etc/default"
-  plat.servicetype "sysv"
+  plat.servicetype "systemd"
   plat.codename "jessie"
 
   plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/debian/pl-build-tools-release-jessie.deb"
```
Comment (melissa) [inline configs/platforms/debian-8-amd64.rb]: since you'd editing this file anyway, wanna pull this line out?

**Comment by mwaggett** [comment]
@melissa Updated again! I removed that line from all the configs and removed EOL platforms while I was at it.

**Comment by mwaggett** [comment]
@demophoon those changes are in! 😊

**Comment by melissa** [comment]
Have you built and tested the stretch release package yet?

**Comment by mwaggett** [comment]
@melissa doing it now

**Comment by demophoon** [comment]
Oops, I probably should have waited for that.. sorry.

**Pull request merged**